timer.js

```diff
@@ -0,0 +1,42 @@
+let timerInterval;
+let countValue = 0;
+const countElement = document.getElementById("count_timer");
+
+document.getElementById("start_btn").addEventListener("click", () => {
+	startCounter();
+});
+
+document.getElementById("pause_btn").addEventListener("click", () => {
+	pauseCounter();
+});
+
+document.getElementById("resume_btn").addEventListener("click", () => {
+	resumeCounter();
+});
+
+document.getElementById("stop_btn").addEventListener("click", () => {
+	stopCounter();
+});
+
+function startCounter() {
+	clearInterval(timerInterval);
+
+	timerInterval = setInterval(() => {
+		countValue++;
+		countElement.textContent = countValue;
+	}, 1000);
+}
```

Comment:
refrain creating a function inside another function.
refer to this guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g2651733a4ec_0_9